index.html

```diff
@@ -1191,6 +1214,37 @@ <h4>Rate-limiting change notifications</h4>
           counters, if not accessed via a higher level framework.
         </p>
       </section>
+      <section>
+        <h4>Rate obfuscation</h4>
+        <p>
+          The specification will recommend the implementation to keep track of the number of
+          pressure changes over an implementation-specific sliding observation window and
+          set a flag if an implementation-specific threshold for the number of pressure
+          changes is exceeded. Similarly, it is also recommended for the implementation to
+          observe any abnormal activity such as a high number of pressure state changes
+          spanning across multiple states, and set this flag similarly.
+        </p>
+        <p>
+          If this flag is set, the implementation is recommended to give the pressure observer
+          a penalty during which it will not be able to inform scripts of changes in its
+          pressure state as it normally would. The duration of this penalty is
+          implementation-defined and it is recommended to be randomized.
+          When [=notify pressure observers=] resumes operation after the penalty, it only
+          reports the latest pressure state and disregards any interim state information
+          received from the platform collector during this penalty.
+        </p>
+      </section>
+      <section>
+        <h4>Break calibration</h4>
+        <p>
+          In a calibration process an attacker tries to manipulate the CPU so that this
+          API would report a transition into a certain pressure state in response to the
+          pressure exerted by the fabricated workload. By slightly changing at runtime
+          the implementation-defined low-level hardware metrics that contribute to these
+          pressure state transitions this mitigation will slow down or prevent this
+          calibration process from succeeding.
+        </p>
+      </section>
```

Comment:
I think we might also want to mention this mitigation: Throttle compute execution of background web app

Browsers already do this in order to save power (sometimes after some time delay like say 5 minutes). Doing this will make it much harder to create workloads that results in certain states, and it might be impossible to reach upper states like "critical" - also the calibrated workloads would also be invalid

Reply:
Fixed by https://github.com/w3c/compute-pressure/pull/219/commits/f77bc3dda4e51ba78d0b4c65cb1e48cd1c380881

I think a note in the context of this mitigation makes sense since this is not specific to this API but a feature of browsers.